pages/rfc/registry-updates.rst

```diff
@@ -27,14 +27,91 @@ DRAFT
 Terminology
 ***********
 
+TODO
+
 - Event: an event in the registry event log, e.g. the RegisterVendor event
 - Event payload: the actual contents of the event, describing an object.
 - Object: the thing an event describes; either a vendor, organization or endpoint.
 - Event header: top-level properties of an event describing its payload.
 
-Challenges
+Operations
 **********
 
+The mutate operations that can be performed (by human operators or systems) on the registry are as follows:
+
+**Vendors**
+
+* Register vendor (currently supported)
+* Update vendor name
+* Reissue CA certificate
+* Rotate keys (and reissue associated certificates)
+* Deactivate/reactivate vendor
```

Comment:
This would be a revocation of the certificate?

Reply:
I like that idea